fw/include/fw/filesystem.hpp

```diff
@@ -1,11 +1,25 @@
 #pragma once
 
-#if 1
+#if defined(_MSC_VER)
+
 #    include <boost/filesystem.hpp>
-namespace filesystem = boost::filesystem;
+
+namespace filesystem
+{
+using namespace boost::filesystem;
+using boost::system::error_code;
+}
+
 #else
+
 #    include <filesystem>
```

Comment:
Does not work on Travis, sadly. Can you re-disable this code? To be picked up separately.

Reply:
Yes I saw this, it is because link to libstdc++fs is missing for filesystem. I will look into this